modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -771,6 +784,67 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
     return this.signRequestBase(params, RequestType.message);
   }
 
+  /**
+   * Get the challenge values for enterprise and BitGo in ECDSA signing
+   * Only returns the challenges if they are verified by the user's enterprise admin's ecdh key
+   * @param bitgo
+   * @param walletId
+   * @param enterpriseId
+   */
+  static async getChallengesForEcdsaSigning(
+    bitgo: BitGoBase,
+    walletId: string,
+    enterpriseId: string
+  ): Promise<ApiChallenges> {
+    const urlPath = `/wallet/${walletId}/challenges`;
+    const result = await bitgo.get(bitgo.url(urlPath, 2)).query({}).result();
```

Comment:
can this be delegated to a function on the wallet class? It's a little odd to do it here.